src/index.mako.html

```diff
@@ -758,6 +758,22 @@
           api_url = my_host;
         }
 
+        var my_shop_url = getParam('shop_url');
```

Comment:
Please avoid to name a variable with `my****` and use camelCase syntax `shopUrl`.